README.md

```diff
@@ -81,6 +81,29 @@ following command:
 npm run test:jest -- -u
 ```
 
+## End-to-End Testing
+
+We use [Playwright](https://playwright.dev/) for our end-to-end tests.
+
+Before testing, download the supported browsers needed for Playwright to execute
+successfully by running:
+
+```bash
+npx playwright install
+```
```

Comment:
adding this step because it's mentioned as part of installation in Playwright docs: https://playwright.dev/docs/intro#installation